src/pages/teachers/programmes/[programmeSlug]/units/[unitSlug]/lessons/[lessonSlug].tsx

```diff
@@ -90,16 +90,18 @@ export const getStaticProps: GetStaticProps<
           notFound: true,
         };
       }
-
-      const { videoTitle } = curriculumData;
-      if (videoTitle && !isSlugLegacy(programmeSlug)) {
+      const { videoTitle, transcriptSentences } = curriculumData;
+      if (videoTitle && !isSlugLegacy(programmeSlug) && !transcriptSentences) {
         // For new content we need to fetch the captions file from gCloud and parse the result to generate
         // the transcript sentences.
         const fileName = `${videoTitle}.vtt`;
         const transcript = await getCaptionsFromFile(fileName);
         if (transcript) {
           curriculumData.transcriptSentences = transcript;
         }
+      } else if (transcriptSentences && !Array.isArray(transcriptSentences)) {
+        const splitTranscript = transcriptSentences.split(/\r?\n/);
```

Comment:
I'm wondering if we could use a more complicated regex here to make the text a little more readable ? New line for new sentence ?

Reply:
agreed, we do something like that in `getCaptionsFromFile`

Reply:
Utilises the formatTranscript function ✅